benchadapt/python/benchadapt/adapters/_adapter.py

```diff
@@ -120,17 +126,23 @@ def post_results(self) -> list:
         """
         Post results of run to conbench
         """
-        client = ConbenchClient()
-
         if not self.results:
             fatal_and_log(
                 "No results attribute to post! Was `run()` called on this instance?"
             )
 
+        log.info("Initializing conbench client")
+        client = ConbenchClient()
+
+        log.info("Posting results to conbench")
         res_list = []
         for result in self.results:
             result_dict = result.to_publishable_dict()
+            log.debug(
+                f"Posting benchmark result to conbench: `{json.dumps(result_dict)}`"
+            )
             res = client.post(path="/benchmarks", json=result_dict)
             res_list.append(res)
```

Comment:
It might be nice to also log a non-failing response here (well in the client.post method really...) too if we've got this level of debugging elsewhere

Reply:
The client does `log.debug("POST <url> <json>")` already, though our default level is set to INFO. I don't see the number of requests here being prohibitively enormous, but it's still probably easier to have users change the logging level to DEBUG if they want to see it all

Reply:
Ah I missed that, great